applications/nrf5340_audio/src/bluetooth/le_audio.h

```diff
@@ -13,20 +13,16 @@
 #define LE_AUDIO_ZBUS_EVENT_WAIT_TIME	  K_MSEC(5)
 #define LE_AUDIO_SDU_SIZE_OCTETS(bitrate) (bitrate / (1000000 / CONFIG_AUDIO_FRAME_DURATION_US) / 8)
 
-#if (CONFIG_AUDIO_SAMPLE_RATE_48000_HZ)
-#define BT_AUDIO_CODEC_CONFIG_FREQ    BT_AUDIO_CODEC_CONFIG_LC3_FREQ_48KHZ
-#define BT_AUDIO_CODEC_CAPABILIY_FREQ BT_AUDIO_CODEC_LC3_FREQ_48KHZ
-#elif (CONFIG_AUDIO_SAMPLE_RATE_24000_HZ)
-#define BT_AUDIO_CODEC_CONFIG_FREQ    BT_AUDIO_CODEC_CONFIG_LC3_FREQ_24KHZ
-#define BT_AUDIO_CODEC_CAPABILIY_FREQ BT_AUDIO_CODEC_LC3_FREQ_24KHZ
-#elif (CONFIG_AUDIO_SAMPLE_RATE_16000_HZ)
-#define BT_AUDIO_CODEC_CONFIG_FREQ    BT_AUDIO_CODEC_CONFIG_LC3_FREQ_16KHZ
-#define BT_AUDIO_CODEC_CAPABILIY_FREQ BT_AUDIO_CODEC_LC3_FREQ_16KHZ
-#endif /* (CONFIG_AUDIO_SAMPLE_RATE_48000_HZ) */
+#define BT_AUDIO_CODEC_CONFIG_FREQ                                                                 \
```

Comment:
Should we remove the "BT" prefix?
- And: is ***_CONFIG_SAMPLE_RATE a better name?

Reply:
This is just a group of BT_AUDIO_CODEC_CONFIG_FREQ_XXKHZ defines, so I thought the name should reflect that, since there is a difference between the CONFIG_FREQ and the CAPABILITY_FREQ